app/dockerdwrapper.c

```diff
@@ -483,19 +485,53 @@ static bool start_dockerd(const struct settings* settings, struct app_state* app
     guint args_offset = 0;
     gchar** args_split = NULL;
 
+    g_autofree char* log_level = get_parameter_value(param_handle, PARAM_DOCKERD_LOG_LEVEL);
+
+    // get host ip
+    char host_buffer[256];
+    char* IPbuffer;
+    struct hostent* host_entry;
+    gethostname(host_buffer, sizeof(host_buffer));
+    host_entry = gethostbyname(host_buffer);
+    IPbuffer = inet_ntoa(*((struct in_addr*)host_entry->h_addr_list[0]));
+
+    // construct the rootlesskit command
+    args_offset += g_snprintf(args + args_offset,
+                              args_len - args_offset,
+                              "%s %s %s %s %s %s %s %s %s",
+                              "rootlesskit",
+                              "--subid-source=static",
+                              "--net=slirp4netns",
+                              "--disable-host-loopback",
+                              "--copy-up=/etc",
+                              "--copy-up=/run",
+                              "--propagation=rslave",
+                              "--port-driver slirp4netns",
+                              /* don't use same range as company proxy */
+                              "--cidr=10.0.3.0/24");
```

Comment:
How can we sure that someone else isn't annoyed by this choice? We could easily make this into a parameter "RootlesskitNetwork" with a default value of "10.0.3.0/24".

Reply:
I think we can keep it as this for now - slirp4netns uses 10.0.2.0 as default so it is not a massive change.
At this point I think it is overkill to add a random settings when we do not know which will be of interest to change.
If someone is annoyed with it they can either patch this themselves or request a setting.